src/registry/artifactory.rs

```diff
@@ -75,6 +75,11 @@ impl Artifactory {
         };
 
         let status = response.status();
+
+        if !status.is_success() {
+            tracing::info!(response_header=?response.headers());
```

Comment:
debug maybe?

Reply:
If we use debug we may need to override RUST_LOG env var in CI - hopefully this should be possible?

Reply:
You might want to consider the [debug string](https://github.com/seanmonstar/reqwest/blob/3abcc7c4f537c16ad9937f8cc60fb23cb506ac85/src/async_impl/response.rs#L389-L397) too:

```suggestion
            tracing::debug!("artifactory response: {:?}", response);
```

Reply:
Yes. You have an example here: https://github.com/helsing-ai/buffrs/blob/3120b1a7bb16d8fd47cfbe98ced5dcfa9635df41/.github/workflows/ci.yml#L34

Reply:
We're trying to fix a bug that only appears in CI, and I'm not sure how to override RUST_LOG there.

Reply:
Regarding `debug!`, I don't see my debug logs when I run this. I seem to remember a lot of custom initialization being done to the:

```bash
RUST_LOG=debug cargo run -- login --registry https://my-registry...
export RUST_LOG=debug; cargo run -- login --registry https://my-registry...
```

(This doesn't change with `RUST_BACKTRACE=1`)